app/views/schools/ects/show.html.erb

```diff
@@ -9,69 +9,12 @@
 
 <%= render Schools::TeacherProfileSummaryListComponent.new(@ect) %>
 
-<h2 class='govuk-heading-m'>ECTE training details</h2>
-<%=
-  govuk_summary_card(title: 'Reported to us by your school') do |card|
-    if @ect.provider_led?
-      card.with_summary_list(
-        rows: [
-          { key: { text: 'Appropriate body' },
-            value: { text: appropriate_body_name(appropriate_body_type: @ect.appropriate_body_type,
-                                                 appropriate_body: @ect.appropriate_body) } },
-          { key: { text: 'Programme type' },   value: { text: programme_type_name(@ect.programme_type) } },
-          { key: { text: 'Lead provider' },    value: { text: @ect.lead_provider_name } },
-        ]
-      )
-    else
-      card.with_summary_list(
-        rows: [
-          { key: { text: 'Appropriate body' },
-            value: { text:appropriate_body_name(appropriate_body_type: @ect.appropriate_body_type,
-                                                appropriate_body: @ect.appropriate_body) } },
-          { key: { text: 'Programme type' },   value: { text: programme_type_name(@ect.programme_type)  } },
-        ]
-      )
-    end
-  end
-%>
+<h2 class="govuk-heading-m">ECTE training details</h2>
+
+<%= render Schools::SummaryCardComponent.new(title: 'Reported to us by your school', ect: @ect, data_source: :school) %>
+
+<%= render Schools::SummaryCardComponent.new(title: 'Reported to us by your lead provider', ect: @ect, data_source: :lead_provider) %>
+
+<%= render Schools::SummaryCardComponent.new(title: 'Reported to us by your appropriate body', ect: @ect, data_source: :appropriate_body) %>
 
-<%=
-  if @ect.provider_led?
```

Comment:
Am I right in thinking that if `@ect.provider_led?` is `false`, `training_periods` wouldn't exist, making the `@ect.provider_led?` check redundant?

- `ect_at_school_period` [can have many](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/b9ebeb4b0394c4778e45a663b5f561b1721ebb14/app/models/ect_at_school_period.rb#L25) `training_periods`.



- Each `training_period` [belongs to](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/models/training_period.rb#L12-L13) a `lead_provider` and `delivery_partner`

Reply:
This is such a good question! @peteryates @tonyheadford 

If Avin is right, then we should really rename training_periods to be reflective of the fact it's provider_led_training_periods, surely?

Reply:
It would mean that if they switch to provider-led then we would lose the info about the time spent doing school-led unless we add an additional `ECTAtSchoolPeriod` which goes against what it represents

Reply:
so maybe the training_period is the right place for it?

Reply:
I guess we'd know because it's the difference between the start_date of the ect_at_school period, and the start date of the first provider_led_training_period.

However, if we still think they're something different, maybe it's still best to separate, but that means a training_period really belongs to an ECT, and sometimes it has a pairing (of LP and DP) associated with it?